examples/llm/src/data/denoising.py

```diff
@@ -815,17 +816,19 @@ def _format_tokens_for_decoder_only(
 
 
 # Helpful to test if your dataloader is working locally
-# Run `python denoising.py [remote] [local, optional]` and verify that batches
+# Run `python denoising.py [local] [remote, optional]` and verify that batches
 # are printed out
 if __name__ == '__main__':
-    remote = sys.argv[1]
+    from examples.common.builders import build_tokenizer
+
+    local = sys.argv[1]
     if len(sys.argv) > 2:
-        local = sys.argv[2]
+        remote = sys.argv[2]
     else:
-        local = remote
+        remote = local
     print(f'Reading val split from {remote} -> {local}')
 
-    decoder_only = True
+    decoder_only = False
```

Comment:
What's wrong with `decoder_only = True`? If we set it to `False`, we need to remove `packing_ratio` from the `cfg` because it won't be supported.

Reply:
sorry I was just seeing if it would still run...
shouldn't have merged this.